frontend/src/component/common/NewConstraintAccordion/ConstraintsList/EditableConstraintsList.tsx

```diff
@@ -29,6 +33,15 @@ export const EditableConstraintsList = forwardRef<
 >(({ constraints, setConstraints }, ref) => {
     const { context } = useUnleashContext();
 
+    useImperativeHandle(ref, () => ({
```

Comment:
Implement addConstraint method for ref.

Reply:
Huh. Interesting. Don't think I've seen us do this kinda thing before. In fact, I didn't know you could 😅 But hey, if it works, it works